core/services/privacy/policy/PrivacyPolicyManager.php

```diff
@@ -26,25 +26,27 @@ class PrivacyPolicyManager
 
     public function __construct()
     {
-        add_action('admin_init', array($this, 'addPrivacyPolicy'), 9);
+        add_action('current_screen', array($this, 'addPrivacyPolicy'), 9);
     }
 
 
+
     /**
      * For all the registered `PrivacyPolicyInterface`s, add their privacy policy content
      *
      * @param WP_Post $post
      */
     public function addPrivacyPolicy()
     {
-        $policy_page_id = (int) get_option('wp_page_for_privacy_policy');
-        if (! $policy_page_id) {
-            return;
-        }
-        // load all the privacy policy stuff
-        // add post policy text
-        foreach ($this->loadPrivacyPolicyCollection() as $privacy_policy) {
-            wp_add_privacy_policy_content($privacy_policy->getName(), $privacy_policy->getContent());
+        $screen = get_current_screen();
+        if ($screen instanceof \WP_Screen
```

Comment:
the `do_action( 'current_screen', $current_screen );` hookpoint passes the global `$current_screen` object as a parameter, so instead of calling `get_current_screen()` you can simply type hint for `WP_Screen` in your callback.

As well, plz import all classes, even those from the root namespace

Reply:
oh right- both good calls